### DISCUSSION TITLE
Footer and Navbar colors

### DISCUSSION THREAD

**Discussion by binHarby** [SpecialProjectADU/Site#101] [Announcements]
@AlyafeiAli  @Moutasim02 

Can we agree on the background and text colors for the navbar and footer?

I wanted to follow the standard for the footer, but the color grey doesn't seem dark enough, and I like the dark color from bootstrap better @Moutasim02 .Lets discuss this, and finalize the colors for both the 'color' and 'background-color' of the navbar and the footer

**Comment by binHarby** [comment]
Also, @AlyafeiAli  could you move the breakpoint transition of your navbar to the `Navbar.css`? because it  broke on Audio.html. When you scroll down, the navbar background doesn't change

**Reply by AlyafeiAli** [reply]
> Also, @AlyafeiAli could you move the breakpoint transition of your navbar to the `Navbar.css`? because it broke on Audio.html. When you scroll down, the navbar background doesn't change

Audio.html is not using the universal navbar aslan lol. I believe Alyazi will fix that with the next push.

**Reply by binHarby** [reply]
> > Also, @AlyafeiAli could you move the breakpoint transition of your navbar to the `Navbar.css`? because it broke on Audio.html. When you scroll down, the navbar background doesn't change
> 
> Audio.html is not using the universal navbar aslan lol. I believe Alyazi will fix that with the next push.

I was actually the one to tell her to copy the navbar from `index.html`, since the starting section looks very similar  `Audio.html` first section. I think it'd look better with the current audio page

**Reply by binHarby** [reply]
But you can tell her to use the other one if you think its better

**Reply by AlyafeiAli** [reply]
Yeah we already told her in #88

**Comment by AlyafeiAli** [comment]
> @AlyafeiAli @Moutasim02
> 
> Can we agree on the background and text colors for the navbar and footer?
> 
> I wanted to follow the standard for the footer, but the color grey doesn't seem dark enough, and I like the dark color from bootstrap better @Moutasim02 .Lets discuss this, and finalize the colors for both the 'color' and 'background-color' of the navbar and the footer

The current one 's color is nice enough imo. I don't see why we need to change its color or the overall design.

**Comment by Moutasim02** [comment]
> @AlyafeiAli @Moutasim02
> 
> Can we agree on the background and text colors for the navbar and footer?
> 
> I wanted to follow the standard for the footer, but the color grey doesn't seem dark enough, and I like the dark color from bootstrap better @Moutasim02 .Lets discuss this, and finalize the colors for both the 'color' and 'background-color' of the navbar and the footer

Current color is fine, just need to fix the texts
